src/layouts/Navbar.jsx

```diff
@@ -11,6 +11,8 @@ import { useSelector } from "react-redux";
 import { NavLink } from "react-router-dom";
 import { useHistory, useLocation } from "react-router-dom";
 import CryptoJS from "crypto-js";
+import { Dropdown, DropdownMenu, DropdownToggle } from "reactstrap";
+import pic from "../../public/assets/BlogPagepics/blog1.png";
```

Comment:
public'den çekerken 
1. import etmene gerek yok

Reply:
2. pic değişkenini kullandığın yerde "./assets/BlogPagepics/blog1.png" yazmalısın

Reply:
Gökhan Bey onu dropdown UI'yı düzenlemek için import etmiştim de dinamik olarak daha sonra reducer'dan aldım.